### PR TITLE
pdksync - (CONT-189) Remove support for RedHat6 / OracleLinux6 / Scientific6

### PR DESCRIPTION
**Pull request by david22swan**
CONT-189/remove_os_support
pdk version: `2.5.0`

### CODE REVIEW

**Review by LukasAud — DISMISSED** [review]

**Review by ekohl — COMMENTED** [review]
I think dropping EL6 implies dropping Apache 2.0 support. Any plans on really ripping that out? It would massively clean up the module.

File: spec/classes/mod/info_spec.rb [hunk 1]
```diff
@@ -140,7 +140,7 @@ def general_info_specs_apache24
   end
 
   context 'on a RedHat OS' do
-    include_examples 'RedHat 6'
+    include_examples 'RedHat 8'
 
     # Load the more generic tests for this context
     general_info_specs_apache22
```
Comment (ekohl) [inline spec/classes/mod/info_spec.rb:146]: This is misleading. EL6 had apache 2.2 but EL8 has 2.4.

File: spec/classes/mod/spec_helper_local.rb [hunk 2]
```diff
@@ -0,0 +1,208 @@
+# frozen_string_literal: true
```
Comment (ekohl) [inline spec/classes/mod/spec_helper_local.rb:1]: Is this intentional? Looks accidental.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: spec/classes/mod/spec_helper_local.rb (hunk 2 above)
Comment (david22swan) [reply spec/classes/mod/spec_helper_local.rb:1]: accidental

File: spec/classes/mod/info_spec.rb (hunk 1 above)
Comment (david22swan) [reply spec/classes/mod/info_spec.rb]: Removed

**Review by LukasAud — APPROVED** [review]

**Comment by david22swan** [comment]
> I think dropping EL6 implies dropping Apache 2.0 support. Any plans on really ripping that out? It would massively clean up the module.

That will be coming in a later PR to remove compatibility.
For now we are just removing support, with the compatibility work scheduled for when we have more time to dedicate to it.

**Review by chelnak — APPROVED** [review]

**Pull request merged**